rust/phoenix-sdk/src/utils.rs

```diff
@@ -0,0 +1,94 @@
+use std::mem::size_of;
+
+use ellipsis_client::EllipsisClient;
+use phoenix::program::{dispatch_market, get_seat_address, MarketHeader};
+use phoenix_seat_manager::instruction_builders::{
+    create_claim_seat_instruction, create_evict_seat_instruction, EvictTraderAccountBackup,
+};
+use solana_program::{instruction::Instruction, pubkey::Pubkey};
+use spl_associated_token_account::{
+    get_associated_token_address, instruction::create_associated_token_account,
+};
+
+pub async fn create_ata_ix_if_needed(
+    client: &EllipsisClient,
+    payer: &Pubkey,
+    trader: &Pubkey,
+    mint: &Pubkey,
+) -> Option<Instruction> {
```

Comment:
why not return a Vec<Instruction>?

Reply:
Create ATA is only one instruction so vec seemed unnecessary

Reply:
That's true, it would simplify `get_maker_setup_instructions_for_market` a little bit. I can make that change

Reply:
Reflected in [ac6379e](https://github.com/Ellipsis-Labs/phoenix-sdk/pull/87/commits/ac6379eb6b7ab136875dbbdbc04683cf11998065)